content/SCALE/SCALEUIReference/SystemSettings/SystemBootScreensSCALE.md

```diff
@@ -62,8 +64,8 @@ Each boot environment on the list includes:
 * **Name** which is the name of the boot entry as it appears in the boot menu.
 * **Active** that indicates which entry boots by default if a boot environment is not active. Activated environment displays **Non/Reboot**.
 * **Date Created** that shows the creation date and time.
-* **Space** shows the boot environment size.
-* **Keep** that indicates whether TrueNAS deletes this boot environment when a system update does not have enough space to proceed.
+* **Space** shows the boot environment size, which can be helpful during storage provisioning procedures.
```

Comment:
In this case not really, it about the small partition used, or the drive allocated just for boot environments not about data or storage provisioning. Kind of a fuzzy area in terms of overall storage in that in large systems a lower-capacity drive is allocated for boot environments but home users or small systems might just create a partition on a drive to provide this storage. Remove this reference.
```suggestion
* **Space** shows the boot environment size.
```